src/container/FormContainer.jsx

```diff
@@ -4,117 +4,128 @@ import { connect } from 'react-redux';
 import FormComponent from '../component/FormComponent.jsx';
 import ReactDOM from 'react-dom';
 import ReCAPTCHA from 'react-google-recaptcha';
-import { Button, Icon, Modal } from 'semantic-ui-react';
 import { setHours, setMinutes } from 'date-fns';
+import { verifiedCaptcha, resetForm, saveForm, editForm, submitForm } from '../actions/Form';
+import ConfirmFormContentComponent from '../component/ConfirmFormContentComponent';
+import PositionFormFinalStep from '../component/PositionFormFinalStep';
```

Comment:
Do ConfirmFormContentComponent and PositionFormFinalStep need the jsx extension also?